client/src/components/StaticPages/About.js

```diff
@@ -190,8 +195,15 @@ const About = () => {
           <Typography variant="h2">Our Team</Typography>
           <Container maxWidth="sm">
             <Typography variant="body1">
-              We are a 100% volunteer-run project. We are part of Hack for LA.
-              Our team includes product managers, researchers, designers,
+              We are a 100% volunteer-run project. We are part of{" "}
+              <a
+                href="https://hackforla.org/"
+                target={"_blank"}
+                rel="noopener noreferrer"
+              >
+                Hack for LA
+              </a>
```

Comment:
I'd make a custom component for `<ExternalLink />` 🤔

Reply:
@hanapotski Since we're using Material-UI, there is a Link component provided by the library. Should we use that one or create a new one? 🤔 If we don't need much customization, it's good to use the Material-UI Link component that we're already using. 😀